regex-assembly/942120.ra

```diff
@@ -3,15 +3,16 @@
 
 ##!+ i
 
+\=\=
 \!\=
 \&\&
 \|\|
+->
 >>
 <<
 >=
 <=
 <>
-<=>
```

Comment:
I wouldn't remove this operator. We have no idea which dialect this could be from.

Reply:
As discussed above, we're already catching this with `<=`.